contracts/lib/SafetyWithdraw.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.6.12;
+import "../Interfaces/ISafetyWithdraw.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract SafetyWithdraw is Ownable, ISafetyWithdraw {
+
+    function withdrawERC20Token(
+        address tokenAddress,
+        address to,
+        uint256 amount
+    ) external override onlyOwner {
+        IERC20(tokenAddress).transfer(to, amount);
```

Comment:
I worry this can be seen as a rug (owner can withdraw any tokens). I guess it has to go through a gov proposal so its not exactly a "rug".

Reply:
Should we do anything about this?